pint/testsuite/test_quantity.py

```diff
@@ -89,6 +89,13 @@ def test_quantity_repr(self):
         self.assertEqual(str(x), '4.2 meter')
         self.assertEqual(repr(x), "<Quantity(4.2, 'meter')>")
 
+    def test_quantity_hash(self):
+        x = self.Q_(4.2, 'meter')
+        x2 = self.Q_(4200, 'millimeter')
+        y = self.Q_(4.2, 'second')
+        self.assertEqual(hash(x), hash(x2))
+        self.assertNotEqual(hash(y), hash(x))
```

Comment:
This isn't actually a requirement of `hash`, so I'm not sure it should be tested here